beacon-chain/core/epoch/epoch_operations_test.go

```diff
@@ -675,3 +669,51 @@ func TestInclusionDistance_NotFound(t *testing.T) {
 		t.Fatal("InclusionDistance should have failed")
 	}
 }
+
+func TestRandaoMix_Ok(t *testing.T) {
+	randaoMixes := make([][]byte, config.LatestRandaoMixesLength)
+	for i := 0; i < len(randaoMixes); i++ {
+		intInBytes := make([]byte, 32)
+		binary.BigEndian.PutUint64(intInBytes, uint64(i))
+		randaoMixes[i] = intInBytes
+	}
+	state := &pb.BeaconState{LatestRandaoMixesHash32S: randaoMixes}
+	tests := []struct {
+		slot      uint64
+		randaoMix []byte
+	}{
+		{
+			slot:      10,
+			randaoMix: randaoMixes[10],
+		},
+		{
+			slot:      2344,
+			randaoMix: randaoMixes[2344],
+		},
+		{
+			slot:      99999,
+			randaoMix: randaoMixes[99999%config.LatestRandaoMixesLength],
+		},
+	}
+	for _, test := range tests {
+		state.Slot = test.slot + 1
+		mix, err := randaoMix(state, test.slot)
+		if err != nil {
+			t.Fatalf("Could not get randao mix: %v", err)
+		}
+		if !bytes.Equal(test.randaoMix, mix) {
+			t.Errorf("Incorrect randao mix. Wanted: %#x, got: %#x",
+				test.randaoMix, mix)
+		}
+	}
+}
+
+func TestRandaoMix_OutOfBound(t *testing.T) {
+	want := fmt.Sprintf(
+		"input randaoMix slot %d out of bounds: %d <= slot < %d",
```

Comment:
do we want %d <= slot < %d or is that the failure condition? Can you specify wanted or received before that?

Reply:
this is wanted, will change `want ` to `wanted` for better clarity